testdata/.env

```diff
@@ -0,0 +1,2 @@
+GLOBAL=global
```

Comment:
please include comments in this file

Reply:
I have added comments clarifying the intention of both variables in this file.